ocaml/xapi/xapi_vm_migrate.ml

```diff
@@ -966,6 +966,22 @@ let migrate_send'  ~__context ~vm ~dest ~live ~vdi_map ~vif_map ~options =
       debug "Mirror failed for VDI: %s" failed_vdi;
       raise (Api_errors.Server_error(Api_errors.mirror_failed,[Ref.string_of vconf.vdi]))
     end;
+
+    if (TaskHelper.is_cancelling ~__context) && not is_intra_pool then
+      begin match e with
+      | Storage_interface.Does_not_exist(_) ->
+        if (Db.is_valid_ref __context vm) then begin
```

Comment:
You are using parentheses again here. They are not strictly needed and the convention is not to use them for simple if-expressions.